packages/ui-components/src/components/text-area/text-area.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	* @prop --height-default: The height of the text area when there's no text
+	* @prop --height-active: The height of the text area when there's text
+	*/
+	--height-default: 21px;
+	--height-active: 84px;
+}
+
+.text-area-container {
+	display: flex;
+	gap: $spacing-2x;
+
+	.left-icon {
+		margin-top: $spacing-2x;
+	}
+
+	.text-area {
+
+		@include kelvin-night-scrollbar();
+		display: flex;
+		flex-direction: column;
+		flex: 1;
+
+		.text-area-wrapper {
+			padding: $spacing-2x $spacing-3x;
+			border: 1px solid kv-color('neutral-7');
```

Comment:
should this border be also defined in a variable?

Reply:
do you mean a scss var or css var/prop? 

I decided not to expose it to keep the styling consistent by not allowing changes to this.